File: src/BreakfastProvider.Api/Models/Responses/PancakeResponse.cs
namespace BreakfastProvider.Api.Models.Responses;

public record PancakeResponse
{
    public Guid BatchId { get; init; }
    public List<string> Ingredients { get; init; } = [];
    public List<string> Toppings { get; init; } = [];
    public DateTime CreatedAt { get; init; }
}

File: java/breakfast-provider-app/src/main/java/io/lemonlion/breakfast/model/response/PancakeResponse.java
package io.lemonlion.breakfast.model.response;

import java.time.Instant;
import java.util.List;
import java.util.UUID;

/** Twin of C# {@code PancakeResponse}. */
public record PancakeResponse(UUID batchId, List<String> ingredients, List<String> toppings, Instant createdAt) {
}
